src/Altinn.Platform/Altinn.Platform.Register/Register/Models/PartyLookup.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Altinn.Platform.Register.Models
+{
+    /// <summary>
+    /// Represents a lookup criteria when looking for a Party. Only one of the properties can be used at a time.
+    /// If none or more than one property have a value the lookup will respond with bad request.
+    /// </summary>
+    public class PartyLookup : IValidatableObject
```

Comment:
Could we define this model in the Altinn.Platform.Models project instead?

Reply:
My thought here is that this model should be exclusive to Register. Maybe even exclusive to the only method currently using it if that was possible.

Reply:
Won't the part of the solution using this endpoint have to parse the response into the same model at some point?

Reply:
That is a good point. I keep forgetting that we have our own client code in C# that could benefit from the class definition.

Reply:
Moving model to correct project.